modules/gitops-argo-cd-properties.adoc

```diff
@@ -38,16 +43,30 @@ a|* `<processors.operation>` - The number of operation processors.
   * `<sharding.dynamicScalingEnabled>` - Enable dynamic scaling of the Argo CD Application Controller component. Use this property to allow the Operator to scale the number of replicas based on the number of clusters the controller component is managing presently. Setting this property to `true` overrides the configuration of the `sharding.enabled` and `sharding.replicas` properties.
   * `<sharding.minShards>` - The minimum number of Argo CD Application Controller replicas.
   * `<sharding.maxShards>` - The maximum number of Argo CD Application Controller replicas.
-  * `<sharding.clusterPerShard>` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
+  * `<sharding.clustersPerShard>` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
   * `<env>` - Environment to set for the application controller workloads.
   * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
 
 |`disableAdmin`    |Disables the built-in admin user.|`false` |
 
+|`defaultClusterScopedRoleDisabled`    |Disables the creation of default `ClusterRoles` for a cluster-scoped instance. | |
+
+|`ExtraConfig`    |Add any supplementary Argo CD settings to the `argocd-cm` ConfigMap that cannot be configured directly within the Argo CD Custom Resource.|`__<empty>__` |
```

Comment:
Fields should always begin with a lowercase letter, so `ExtraConfig` -> `extraConfig`